### PR TITLE
Fixes grouping of sessions by days

### LINKED ISSUE #20: Wizard groups sessions using UTC date
## Symptom

The available times shown in the wizard are incorrect, when the configured timezone is not UTC.

## Steps to Reproduce

1. Set up a service with a timezone that is significantly offset from UTC (such as `America/Los_Angeles`) with a simple daily availability (for example, from 13:00 to 22:00 and 1 hour long sessions).
2. Allow the timezone selector to be shown in the wizard.
3. Select an available date in the wizard.
4. Observe symptom: displayed available times are incorrect.

## Possible Cause

The wizard groups sessions using their UTC start datetimes, but it should groups sessions using their timezone-relative start datetimes.

### Recommended Solution

Apply the configured timezone to the session start times before grouping.

### PR DESCRIPTION
**Pull request by romalytvynenko**
fixes #20

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,13 +53,15 @@ <h1>LOADING...</h1>
                 ></session-duration-picker>
             </div>
             <div>
+                <div><code>OPENED ON DATE: {{ openedOnDate}}</code></div>
```
Comment (XedinUnknown) [inline index.html]: i18n

Comment (romalytvynenko) [reply index.html]: @XedinUnknown, this is dummy page just for development purposes. We don't need i18n here.

**Comment by romalytvynenko** [comment]
@XedinUnknown, because it appears that date picker doesn't have custom `timezone` support. And it works much predictable when you are giving only day string to it, not `Date`. `Date` object holds timezone information, and `YYYY-MM-DD` string doesn't.

So this is what I've done: I'm doing all timezone manipulations before touching that 3-rd party component. When I need to interact with it, I'm giving date string to it.

**Comment by XedinUnknown** [comment]
I'd say, there is a hidden problem somewhere here, which we do not see yet, and which will cause trouble later.

**Comment by romalytvynenko** [comment]
I think that the problem is using `Date` object. Strings much more predictable than `Date`.

**Comment by XedinUnknown** [comment]
The problem is not in `Data` VS string. The problem is in the handling of timezone information. Right now, it may work, but I feel that you're not sure why.

**Pull request merged**